main.go

```diff
@@ -16,15 +18,32 @@ import (
 )
 
 func main() {
+
+	configFlag := flag.String("config", "./config.json", "File to read configuration")
```

Comment:
Could we default to ./data/config.json? We want the sub folder so that we can mount a volume for docker.